integrations/slack/src/router.ts

```diff
@@ -68,12 +68,23 @@ export const handleFetchEvent: FetchEventCallback = async (request, context) =>
     );
 
     // event triggers, e.g app_mention
-    router.post('/events', verifySlackRequest, acknowledgeSlackRequest);
+    router.post(
+        '/events',
+        verifySlackRequest,
+        createSlackEventsHandler(
```

Comment:
It's slightly confusing that we handle both `/events` and `/events_task`. Maybe for another PR, but what do you think of something like this

```
 router.post(
        '/events',
        verifySlackRequest,
        acknowledgeSlackRequest // which can handle url verification
)
        
```

```
 router.post(
        '/events_task',
        verifySlackRequest,
        createSlackEventsHandler(...) // concerned with handling logic rather than ack
```

Reply:
hmm... I had the the opposite thought. thought it was confusing to be handling routing in two places in two different ways. I think doing that would hide the url_verification rather than follow the same pattern.